configurations/pipeline_config.json

```diff
@@ -31,14 +36,20 @@
     {"RapidProKey": "Current_Practices (Value) - icraf_s01_follow_up_survey", "PipelineKey": "current_practices_raw"},
     {"RapidProKey": "New_Practices (Value) - icraf_s01_follow_up_survey", "PipelineKey": "new_practices_raw"},
     {"RapidProKey": "Organizations (Value) - icraf_s01_follow_up_survey", "PipelineKey": "organizations_raw"},
-    {"RapidProKey": "Reasons_For_New_Practices (Value) - icraf_s01_follow_up_survey", "PipelineKey": "new_practice_challenge_raw"},
-    {"RapidProKey": "Upper_Tana_Practices (Value) - icraf_s01_follow_up_survey", "PipelineKey": "upper_tana_practices_reasons_raw"},
+    {"RapidProKey": "Reasons_For_New_Practices (Value) - icraf_s01_follow_up_survey", "PipelineKey": "new_practices_challenge_raw"},
```

Comment:
Do you mean new_practices_challenge*s*_raw?

(I don't know, just checking!)

Reply:
Thats right because one may answer more than one challenge. Updated!